tests/automation_framework/data/work_order/workordersubmit_encryptedRequestHash_norequesterSignature.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
the file name says encryptedRequestHash_norequesterSignature but you are only removing requesterSignature and doing nothing to RequestHash, can you change the file name?